### ISSUE TITLE
Screen reader accessibility mode reads terminal contents character by character

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#172204]
Testing #171862

Would be nice if for e.g. it read my prompt (which includes my CWD) word by word, delimited by path separator.

**Comment by meganrogge** [comment]
The idea is that the screen reader user will select whatever text they'd like to have read and when the cursor is moved, it will announce the char as you describe. @jooyounseo, is that your expectation as well?

**Comment by jooyoungseo** [comment]
I am very sorry, but I cannot fully understand what @joyceerhl was asking here. Screen reader reads aloud live event as a whole chunk and character-by-character or word-by-word reading (by design) requires users' manual cursor operation.

**Comment by Tyriar** [comment]
> Would be nice if for e.g. it read my prompt (which includes my CWD) word by word, delimited by path separator.

It's up to the screen reader how it reads the text, we can't (shouldn't) change this. We are throwing around an idea of adding semantics to the buffer which could potentially even lead to us showing a cwd field instead of just raw text which aligns with what you're after.

Closing as designed for now
